source/_posts/2024-11-18-event-wrapup-github-universe-24.markdown

```diff
@@ -0,0 +1,86 @@
+
+<img src='/images/blog/2024-11-universe/octoverse-contributors.png' style='border: 0;box-shadow: none;' alt="top five projects by contributors - source GitHub Octoverse 2024 report">
+
+Our friends at GitHub did a great job of making sure we were surprised by this news---and believe me when I say we lost our collective minds when they revealed it to us. 😎 We also found out that we've won GitHub's [Wonderfully Welcoming Award](https://github.blog/news-insights/company-news/celebrating-the-github-awards-2024-recipients/#wonderfully-welcoming-award), which recognizes how we attract and support new contributors to our project.
+
+<img src='/images/blog/2024-11-universe/wonderfully-welcoming.png' style='border: 0;box-shadow: none;' alt="Wonderfully welcoming award">
+
+## Open Source - ⭐️ of the show
+
+Paulus and Frenck were at keynote rehearsals Friday, Sunday, and Monday to be prepared for Wednesday.  So I picked up my badge Monday morning, which was a hackable board with an e-ink display programmed to show our names - the little details they put into this event were quite pleasing. 🤩
+
+There was a community session where GitHub was able to show off a little of what they would be revealing on Day 2. At the breakout session, I was able to share our pain points about moderating a community directly on GitHub, and it was good to hear how other projects do this on the platform.
+
+<p class='img'><img src='/images/blog/2024-11-universe/day-1-keynote.jpg' alt="Paulus, Frenck and myself at the Day 1 keynote">Paulus, Frenck and myself at the Day 1 keynote</p>
+
+Day 1 is when we had our booth in the [Open Source Zone](https://github.blog/open-source/10-projects-in-the-open-source-zone-at-github-universe-2024/), but I was also set to represent Home Assistant at a panel. The session was called [Cha-ching! How to attract funding for your open source projects](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715376172449001OEqW); I was very excited to tell our story since Home Assistant is funded by y'all - the community - through our [Home Assistant Cloud subscription](/cloud/), and not investors. I was pleasantly surprised to hear that others on the panel shared our approach of building a sustainable project through community support! I had great conversations with several people working on other open source projects like Homebrew and Ladybird.
```

Comment:
```suggestion
Day 1 is when we had our booth in the [Open Source Zone](https://github.blog/open-source/10-projects-in-the-open-source-zone-at-github-universe-2024/), but I was also set to represent Home Assistant at a panel. The session was called [Cha-ching! How to attract funding for your open source projects](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715376172449001OEqW); I was very excited to tell our story since Home Assistant is funded by y'all - the community - through our [Home Assistant Cloud subscription](/cloud/), and not investors. I was pleasantly surprised to hear that others on the panel shared our approach to building a sustainable project through community support! I had great conversations with several people working on other open source projects like Homebrew and Ladybird.
```